doc/dg/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+add_custom_target(dg
+        COMMAND ${CMAKE_BINARY_DIR}/doc/latexit.sh
+        ${CMAKE_CURRENT_SOURCE_DIR} dg
+        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
```

Comment:
Don't need `${CMAKE_CURRENT_BINARY_DIR}` here, `add_custom_target`'s `WORKING_DIRECTORY` is relative to the current binary directory, so either use `.` or leave out the parameter.